spark/dl/src/main/scala/com/intel/analytics/bigdl/models/lenet/LeNet5.scala

```diff
@@ -50,6 +50,6 @@ object LeNet5 {
     val fc2 = Linear(100, classNum).setName("fc2").inputs(tanh3)
     val output = LogSoftMax().inputs(fc2)
 
-    Graph(input, output)
+    if(isDynamic) Graph.dynamic(input, output) else Graph(input, output)
```

Comment:
Why do we need isDynamic for the models/examples? It should always be static graph, no?

Reply:
This is for test... We have a set of test for these graph models, so I want to test both dynamic and static